models/mobilinetssd/FP32/mobilenetssd.xml

```diff
@@ -0,0 +1,6838 @@
+		<edge from-layer="358" from-port="4" to-layer="359" to-port="0"/>
+		<edge from-layer="2" from-port="0" to-layer="360" to-port="0"/>
+		<edge from-layer="360" from-port="1" to-layer="361" to-port="0"/>
+		<edge from-layer="361" from-port="1" to-layer="365" to-port="0"/>
+		<edge from-layer="362" from-port="1" to-layer="365" to-port="1"/>
+		<edge from-layer="363" from-port="1" to-layer="365" to-port="2"/>
+		<edge from-layer="364" from-port="1" to-layer="365" to-port="3"/>
+		<edge from-layer="365" from-port="4" to-layer="366" to-port="0"/>
+		<edge from-layer="359" from-port="1" to-layer="367" to-port="0"/>
+		<edge from-layer="366" from-port="1" to-layer="367" to-port="1"/>
+		<edge from-layer="367" from-port="2" to-layer="369" to-port="0"/>
+		<edge from-layer="368" from-port="1" to-layer="369" to-port="1"/>
+		<edge from-layer="223" from-port="2" to-layer="370" to-port="0"/>
+		<edge from-layer="370" from-port="1" to-layer="371" to-port="0"/>
+		<edge from-layer="371" from-port="1" to-layer="375" to-port="0"/>
+		<edge from-layer="372" from-port="1" to-layer="375" to-port="1"/>
+		<edge from-layer="373" from-port="1" to-layer="375" to-port="2"/>
+		<edge from-layer="374" from-port="1" to-layer="375" to-port="3"/>
+		<edge from-layer="375" from-port="4" to-layer="376" to-port="0"/>
+		<edge from-layer="2" from-port="0" to-layer="377" to-port="0"/>
+		<edge from-layer="377" from-port="1" to-layer="378" to-port="0"/>
+		<edge from-layer="378" from-port="1" to-layer="382" to-port="0"/>
+		<edge from-layer="379" from-port="1" to-layer="382" to-port="1"/>
+		<edge from-layer="380" from-port="1" to-layer="382" to-port="2"/>
+		<edge from-layer="381" from-port="1" to-layer="382" to-port="3"/>
+		<edge from-layer="382" from-port="4" to-layer="383" to-port="0"/>
+		<edge from-layer="376" from-port="1" to-layer="384" to-port="0"/>
+		<edge from-layer="383" from-port="1" to-layer="384" to-port="1"/>
+		<edge from-layer="384" from-port="2" to-layer="386" to-port="0"/>
+		<edge from-layer="385" from-port="1" to-layer="386" to-port="1"/>
+		<edge from-layer="301" from-port="2" to-layer="387" to-port="0"/>
+		<edge from-layer="318" from-port="2" to-layer="387" to-port="1"/>
+		<edge from-layer="335" from-port="2" to-layer="387" to-port="2"/>
+		<edge from-layer="352" from-port="2" to-layer="387" to-port="3"/>
+		<edge from-layer="369" from-port="2" to-layer="387" to-port="4"/>
+		<edge from-layer="386" from-port="2" to-layer="387" to-port="5"/>
+		<edge from-layer="230" from-port="2" to-layer="388" to-port="0"/>
+		<edge from-layer="284" from-port="2" to-layer="388" to-port="1"/>
+		<edge from-layer="387" from-port="6" to-layer="388" to-port="2"/>
+		<edge from-layer="388" from-port="3" to-layer="389" to-port="0"/>
+	</edges>
+	<meta_data>
+		<MO_version value="0.0.0-2893-g24aea238ee"/>
+		<cli_parameters>
+			<blobs_as_inputs value="True"/>
+			<caffe_parser_path value="DIR"/>
+			<data_type value="FP16"/>
```

Comment:
Is this a FP16 model? Should we put it under `/FP16` folder?

Reply:
And the model folder name has a typo, `mobilinetssd` -> `mobilenetssd`?

Reply:
Fixed in commit: "Fixed getting output dims. Fix typos in model name"

Reply:
sorry. I didn't find the new commit "Fixed getting output dims. Fix typos in model name", did I miss anything?